playbooks/olm/private/config.yml

```diff
@@ -20,7 +20,7 @@
   tasks:
   - import_role:
       name: olm
-    when: openshift_version | version_compare('3.9', '>=')
+    when: openshift_version is version_compare('3.9', '>=')
```

Comment:
according to the docs, version_compare was renamed to version in 2.5

Reply:
Done, pushed a commit to rename other instances of this